api/swagger-spec/v1.json

```diff
@@ -17624,6 +17628,40 @@
      }
     }
    },
+   "v1.AzureDiskVolumeSource": {
+    "id": "v1.AzureDiskVolumeSource",
+    "description": "AzureDisk represents an Azure Data Disk mount on the host and bind mount to the pod.",
+    "required": [
+     "diskName",
```

Comment:
Better yet, the Portal doesn't show the value that you need to use for "diskName" anywhere (or else it does and **incorrectly** appends a ".vhd" suffix. I'm going to start an internal thread about this issue.